spec/integration/dry/configurable/setting_spec.rb

```diff
@@ -398,6 +398,14 @@
       expect(object_2.config.after).to eq "defined after"
     end
 
+    it "is frozen when finalized" do
+      object.finalize!
+      object.finalize! # second call becomes a no op
+
+      expect(object).to be_frozen
+      expect(object.config).to be_frozen
+    end
```

Comment:
This test just makes it clearer that calling finalize will freeze the object itself (as well as config) when `Dry::Configurable` is included.